### ISSUE TITLE
UABE 3.0 Beta 1 Marked as Dangerous

### ISSUE THREAD

**Issue by Autarch-Conne** [SeriousCache/UABE#586]
When I attempt to download the file my browser detects the file as a virus. Is this something I should take seriously, or just a misfire?

**Comment by Wobbley** [comment]
Getting the same thing.

**Comment by Raccoon-png** [comment]
> When I attempt to download the file my browser detects the file as a virus. Is this something I should take seriously, or just a misfire?

It's a misfire. If you're dealing with windows refusing to open the file or outright deleting it, try downloading an earlier release. I downloaded the earliest release and my browser/windows didn't flag it as a virus. It also worked perfectly for what I needed it for.

**Comment by MacoDeVinx** [comment]
> > When I attempt to download the file my browser detects the file as a virus. Is this something I should take seriously, or just a misfire?
> 
> It's a misfire. If you're dealing with windows refusing to open the file or outright deleting it, try downloading an earlier release. I downloaded the earliest release and my browser/windows didn't flag it as a virus. It also worked perfectly for what I needed it for.

Yeah the 2.2 UABE download works fine, but the 3.0 Beta comes out as a virus for some reason (Months ago it worked without a virus detection)

**Comment by Gr0nal** [comment]
I need support for Unity 2020.3 which 2.2 doesn't, right? 
Trying to work out how to download this cos it just gets straight up deleted whichever browser I use

**Comment by mikedg1** [comment]
If your download is getting deleted immediately you should turn Windows Defender Real Time protection off. Keep in mind that while this is probably a false positive. it's isn't unheard of for a build server to be compromised and have malware injected unbeknownst to the developer.

![image](https://user-images.githubusercontent.com/143783/191990081-a6461a8a-c2fa-42b3-a2df-007992d3ca41.png)

**Comment by Gr0nal** [comment]
Even with Defender real time protection off, it still does it. I can't work out why it's not letting me download it.

**Comment by thecrazy** [comment]
A misfire on 28 different AV? Fix your shit.

![image](https://user-images.githubusercontent.com/2411956/197324847-3f8f408b-24ab-4f52-a9fd-a31a9dc3f696.png)

**Comment by enginsoft** [comment]
> > When I attempt to download the file my browser detects the file as a virus. Is this something I should take seriously, or just a misfire?
> 
> It's a misfire. If you're dealing with windows refusing to open the file or outright deleting it, try downloading an earlier release. I downloaded the earliest release and my browser/windows didn't flag it as a virus. It also worked perfectly for what I needed it for.

it is not a misfire Here is the file scanned in virus total https://www.virustotal.com/gui/file/61365363c523e161d65f0ee91a9ac995ca7194f647a0e5efc13b490273293d96/detection

**Comment by Ecorous** [comment]
I'm going to post this comment as hopefully a last message for people reading these threads. Please don't post anything after this unless it is useful to others reading these threads.

UABE does in fact appear to be a virus, so I would avoid using it for now. I would suggest using [UABEA](<https://github.com/nesrak1/UABEA/releases/latest>) which was and is a valid replacement before UABE was updated and/or opensourced. UABEA may (and is) still buggy, so be aware of that. Again I would entirely stay away from UABE as there has been no proof that it is not a virus, where **many** pieces of antivirus software (including Windows Defender) pick it up as malicious. Until this has been resolved, stay well clear of UABE, and if you need a replacement, use [UABEA](<https://github.com/nesrak1/UABEA/releases/latest>)

**Comment by MacoDeVinx** [comment]
Actually, I would recommend 2.2 b or other 2.2 version of [UABE](https://github.com/SeriousCache/UABE/releases/tag/2.2stabled).
It is only the new UABE version [3.0 beta 1] that has virus detection on it.

Personally not a fan of the UABEA layout and it's also missing a few key features for modding, such as mesh modding and audio files, the only thing you can do is textures.

By using 2.2 you can do whatever but only for games with Unity 2019.2 though.

[Follow up on Ecorous message.]
